pkgs/by-name/po/poutine/package.nix

```diff
@@ -15,13 +16,22 @@ buildGoModule rec {
     hash = "sha256-T81Qi79VaZzfKL4niTZQW+gwwiBcyInALrvyUg1V4Ck=";
   };
 
+  nativeBuildInputs = [ installShellFiles ];
+
   vendorHash = "sha256-/chq40j+puAI3KdI15vbZwrnzCKrU7g+Z/t9aOXQ1Sg=";
 
   ldflags = [
     "-s"
     "-w"
   ];
 
+  postInstall = ''
+    installShellCompletion --cmd ${meta.mainProgram} \
+      --bash <($out/bin/${meta.mainProgram} completions bash) \
```

Comment:
```
poutine> Error: unknown command "completions" for "poutine"
poutine> Did you mean this?
poutine>        completion
poutine> Run 'poutine --help' for usage.
```

completions -> completion indeed seems to fix it

Reply:
Ah, thanks for catching that... I lazily checked the expected files were there but not the fact they were empty.

Reply:
This will be an error soon! https://github.com/NixOS/nixpkgs/pull/289517 Because I also messed up the same thing before :)